src/components/GroupTable/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "./style.css";
+
```

Comment:
we should converge over the next week or so on a particular style guide, personally I like to organize imports like this:

```
import React from "react"; 
import FacebookGroup from "../../models/FacebookGroup";
import "./style.css";
```

I do 3rd party library imports first by order of how "foundational" they are (like react at the top, some minor UI lib at the bottom) followed by relative imports which are ordered by proximity as well as file type (css and data at the bottom).